src/components/ui/LinkedWalletWarning.tsx

```diff
@@ -0,0 +1,37 @@
+import { AccentText } from 'components/ui/Text'
+import Mark, { MarkType } from 'components/ui/Mark'
+import ToolTip from 'components/ui/ToolTip'
+import classnames, {
+  alignItems,
+  display,
+  gap,
+  margin,
+} from 'classnames/tailwind'
+
+const hintWrapper = classnames(
+  display('inline-flex'),
+  alignItems('items-center'),
+  gap('gap-x-1.5'),
+  margin('mt-3')
+)
+
+export default function () {
+  return (
+    <span>
+      <AccentText color="text-error">
+        <ToolTip
+          position="bottom"
+          fitContainer
+          text={
```

Comment:
Let's remove `{}` from here, eslint will come soon for this rule :)

Reply:
ty